### PR TITLE
kubectl: fix the Scheduled eventTime is <unknown> when printing event

### PR DESCRIPTION
**Pull request by gosoon**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
When scheduler using v1beta1.events([#78447](https://github.com/kubernetes/kubernetes/pull/78447)),the `Scheduled` event is not set the LastTimestamp and the FirstTimestamp,only set the EventTime,so use `kubectl get event and kubectl describe pod xxx` you will see that the timestamp of the `scheduled` event is `<unknown>`,as shown below,in those cases we fallback to eventTime.

```
$ kubectl get event
LAST SEEN   TYPE     REASON              OBJECT                        MESSAGE
<unknown>   Normal   Scheduled           pod/nginx-6c975b59f8-gvmjr    Successfully assigned default/nginx-6c975b59f8-gvmjr to minikube

$ kubectl describe pod xxx
......
Events:
  Type    Reason     Age        From               Message
  ----    ------     ----       ----               -------
  Normal  Scheduled  <unknown>  default-scheduler  Successfully assigned default/nginx-6c975b59f8-gvmjr to minikube
......
```

**Which issue(s) this PR fixes**:

Fixes # [89689](https://github.com/kubernetes/kubernetes/issues/89689)

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Fix kubectl printer to correctly handle timestamps of events emitted using events.k8s.io API
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Comment by zhouya0** [comment]
Thanks for your PR, but seems this is dup of https://github.com/kubernetes/kubernetes/pull/89999 which is still waiting for another review.

**Comment by gosoon** [comment]
Ok, I will resubmit a pr only to fix `kubectl describe pod xxx` part.

File: staging/src/k8s.io/kubectl/pkg/describe/describe.go
Comment (zhouya0) [inline staging/src/k8s.io/kubectl/pkg/describe/describe.go:3646]: `EventTime` is `metav1.MicroTime` but not `metav1.Time`. So you need another `translateMicroTimestampSince `. Like https://github.com/kubernetes/kubernetes/pull/89999

Comment (gosoon) [reply staging/src/k8s.io/kubectl/pkg/describe/describe.go:3646]: done

Comment (gosoon) [reply staging/src/k8s.io/kubectl/pkg/describe/describe.go]: done

**Review by soltysh — APPROVED** [review]
/approve
/priority backlog
/retest

**Comment by dmcnaught** [comment]
We're seeing this issue in one of our 1.16.8 clusters. Can we get the fix added to a 1.16 point release?

**Comment by dmcnaught** [comment]
@gosoon We're having this problem with one of our 1.16 clusters and are wondering when/if there will be a 1.16 patch release to fix it. Do you know?

**Comment by cpanato** [comment]
@dmcnaught Hello Duncan, 1.16 is no longer receiving the patch releases, I recommend you to upgrade your cluster to 1.17 at least.
The current supported Kubernetes versions are 1.17 / 1.18 and 1.19

**Comment by dmcnaught** [comment]
> @dmcnaught Hello Duncan, 1.16 is no longer receiving the patch releases, I recommend you to upgrade your cluster to 1.17 at least.
> The current supported Kubernetes versions are 1.17 / 1.18 and 1.19

yes - we upgraded to 1.17 in July/August.

There was a PR in our PaaS that helps up around the unknown timestamps: https://github.com/teamhephy/controller/pull/129

**Pull request merged**
